CONTRIBUTING.md

```diff
@@ -1,19 +1,8 @@
-## Contributing to App Services SDK
-
-## Regenerating all SDKs locally
-
-In some situations like OpenAPIGenerator update or update of the common parts of the code.
-To regenerate all SDK's locally please execute:
-
-```
-./scripts/generate_all.sh
```

Comment:
Since `generate_all` does not currently work, I have removed it. We can revisit how to do this in the future if needed. Since the client payload is unknown to the SDK we could also need to hard-code payloads somewhere in this repo.

Reply:
Amazing

Reply:
Actually so we losing ability to regenerate things locally?
Why not apply js-sdk approach to regenerate all?